libs/feature/map/src/lib/map-context/map-context.service.ts

```diff
@@ -123,22 +124,21 @@ export class MapContextService {
   }
 
   createView(viewModel: MapContextViewModel, map?: Map): View {
-    if (viewModel.extent && map) {
-      const { center, zoom } = this.mapUtils.getViewFromExtent(
-        viewModel.extent,
-        map
-      )
-      viewModel = { ...viewModel, center, zoom }
-    }
     const { center, zoom, maxZoom, maxExtent } = viewModel
-    return new View({
+    const view = new View({
       center,
       zoom,
       maxZoom,
       extent: maxExtent,
       multiWorld: false,
       constrainResolution: true,
     })
+    if (viewModel.extent && map) {
+      view.fit(viewModel.extent, {
```

Comment:
I feel like we originally used `this.mapUtils.getViewFromExtent` instead of `view.fit` for a reason byt I can't remember why... @tkohr do you remember?